### PR TITLE
Fix/ fileinfo service name

### PR DESCRIPTION
**Pull request by bogdanfazakas**
Fixes # .

Changes proposed in this PR:

- 
- 
-

### CODE REVIEW

**Review by jamiehewitt15 — APPROVED** [review]
Looks good to me

File: src/components/httpRoutes/routeUtils.ts [hunk 1]
```diff
@@ -97,7 +97,7 @@ routesNames.set('broadcastCommand', {
   method: 'post'
 })
 // fileinfo
-routesNames.set('fileInfo', {
+routesNames.set('fileinfo', {
   path: '/api/fileinfo',
```
Comment (mariacarmina) [inline src/components/httpRoutes/routeUtils.ts:101]: change path please

**Review by mariacarmina — COMMENTED** [review]
after let's merge

File: src/components/httpRoutes/routeUtils.ts (hunk 1 above)
Comment (bogdanfazakas) [reply src/components/httpRoutes/routeUtils.ts:101]: yep, good catch, need to update the http method itself since it was not backwards compatible

Comment (mariacarmina) [reply src/components/httpRoutes/routeUtils.ts:101]: it is updated in https://github.com/oceanprotocol/ocean-node/pull/485/commits/2242aca9569e48a87c4a5faad74eb2df7e5f027d

**Review by mariacarmina — APPROVED** [review]

**Comment by paulo-ocean** [comment]
we need to have some kind of standard convention for this... all the other routes are Camel case, and this is the only one working differently :-)

**Comment by jamiehewitt15** [comment]
@paulo-ocean Yeah, I agree

**Pull request merged**